memtable.cc

```diff
@@ -737,16 +737,14 @@ memtable::make_flat_reader(schema_ptr s,
     }
 }
 
-flat_mutation_reader
```

Comment:
Commitlog should mention that only the facade is changed, the existing reader is returned via an upgrade.